src/utils/detectors/chrome.js

```diff
@@ -44,16 +46,21 @@ export default class ChromeDetector {
     if (customChromePath) {
       installations.push(customChromePath)
     }
-    execSync(
-      `${LSREGISTER} -dump` +
-        ' | grep -E -i \'(google chrome( canary)?|chromium).app$\'' +
-        ' | awk \'{$1=""; print $0}\''
-    )
+    execSync(`
+      ${LSREGISTER} -dump
+      | grep -E -i '(google chrome( canary)?|chromium)\\.app(\\s|$)'
```

Comment:
If we put exact hex pattern here, we may not need matching in foreach

Reply:
But grep will return whole line anyway. Unless we pass `-o` like you did but then the fix starts to look much like yours :)

Reply:
I've simplified it and now it looks a lot like your fix :)